### DISCUSSION TITLE
Version 24.8 failed to insert data using the new json

### DISCUSSION THREAD

**Discussion by qiuxuhong** [ClickHouse/ClickHouse#73441] [Q&A]
linux system SUSE Linux enterprise server 12 sp5 , linux 4.12.14-122.186-default
create table:
Set allow_experimental_json_type = 1;
create table example (id String, j JSON) engine = MergeTree() order by (id);

insert data:
insert into example(id,j) values('1','{"aa":1}');

Prompt exception:
Cannot parse JSON object here: {"aa":1}: while executing 'FUNCTION if(isNull(-dummy-0) : 3, defaultValueOfTypeName('JSON') :: 2, _CAST(-dummy-0, 'JSON') :: 4) -> if(isNull(-dummy-0), defaultValueOfTypeName('JSON'), _CAST(-dummy-0, 'JSON')) JSON : 1': While executing ValuesBlockInputFormat: data for INSERT was parsed from query. (INCORRECT_DATA) (version 24.8.8.17 (official build))

Please help check what is the problem, is it the bug of clickhouse 24.8.8.17 ?

**Comment by den-crane** [comment]
it works in 24.8.8.17  https://fiddle.clickhouse.com/722617a9-51ad-4086-a468-5f9ff3e767a9
but better use the latest version 24.11.1.2557 https://fiddle.clickhouse.com/722617a9-51ad-4086-a468-5f9ff3e767a9 
because JSON type is experimental and its fixes may be not backported into LTS.
